pkg/storage/config/schema_config.go

```diff
@@ -369,17 +369,17 @@ func (cfg *PeriodConfig) VersionAsInt() (int, error) {
 
 // PeriodicTableConfig is configuration for a set of time-sharded tables.
 type PeriodicTableConfig struct {
-	Prefix string
-	Period time.Duration
-	Tags   Tags
+	Prefix string        `yaml:"prefix" doc:"description=Table prefix for all period tables."`
+	Period time.Duration `yaml:"period" doc:"description=Table period."`
+	Tags   Tags          `yaml:"tags" doc:"description=A map to be added to all managed tables."`
 }
 
 // UnmarshalYAML implements the yaml.Unmarshaler interface.
 func (cfg *PeriodicTableConfig) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	g := struct {
-		Prefix string         `yaml:"prefix"`
-		Period model.Duration `yaml:"period"`
-		Tags   Tags           `yaml:"tags"`
+		Prefix string
```

Comment:
when we will parse the yaml file, we will expect properties starting from capital "P". I believe that it will brake the parsing process....

Reply:
Great catch @vlad-diachenko this completely breaks the parsing. Removed this change in commit: https://github.com/grafana/loki/pull/7981/commits/f585d81c3a6bdce2e58846c85d60c39ea5e24908